graphistry/tests/test_feature_utils.py

```diff
@@ -1,12 +1,18 @@
 # python -m unittest
+import datetime as dt
+import graphistry
+import logging
```

Comment:
@silkspace are you using some tool that is breaking your formatting, causing these spurious (and convention-breaking) changes?